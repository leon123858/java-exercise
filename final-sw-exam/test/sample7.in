student 001 002 003 004
schoolStrategy Excellent,3 Competent,2 NeedsWork,1
designCriterion A1 criteria1_A1.txt
designCriterion A2 criteria1_A1.txt
assignment A1 001 002,score1_1.txt 003,score1_1.txt 004,score1_2.txt
assignment A1 002 001,score1_1.txt 003,score1_1.txt 004,score1_2.txt
assignment A1 003 001,score1_1.txt 002,score1_1.txt 004,score1_2.txt
assignment A1 004 001,score1_1.txt 002,score1_1.txt 003,score1_2.txt
assignment A2 004 001,score1_1.txt 002,score1_1.txt 003,score1_2.txt
assignment A2 003 001,score1_1.txt 002,score1_1.txt 004,score1_2.txt
assignment A2 002 001,score1_1.txt 003,score1_1.txt 004,score1_2.txt
assignment A2 001 002,score1_1.txt 003,score1_1.txt 004,score1_2.txt
averageCriterion A1
calculateScore A1 001 MeanRankingStrategy
findStrength A1 002 MeanRankingStrategy
findWeakness A1 003 MeanRankingStrategy
averageCriterion A2
calculateScore A2 004 MeanRankingStrategy
findStrength A2 002 MeanRankingStrategy
findWeakness A2 001 MeanRankingStrategy