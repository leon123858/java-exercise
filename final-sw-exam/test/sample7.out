A1-001 was reviewed by 002. Level: Excellent Excellent Competent Competent
A1-001 was reviewed by 003. Level: Excellent Excellent Competent Competent
A1-001 was reviewed by 004. Level: Competent Competent Competent Competent
A1-002 was reviewed by 001. Level: Excellent Excellent Competent Competent
A1-002 was reviewed by 003. Level: Excellent Excellent Competent Competent
A1-002 was reviewed by 004. Level: Competent Competent Competent Competent
A1-003 was reviewed by 001. Level: Excellent Excellent Competent Competent
A1-003 was reviewed by 002. Level: Excellent Excellent Competent Competent
A1-003 was reviewed by 004. Level: Competent Competent Competent Competent
A1-004 was reviewed by 001. Level: Excellent Excellent Competent Competent
A1-004 was reviewed by 002. Level: Excellent Excellent Competent Competent
A1-004 was reviewed by 003. Level: Competent Competent Competent Competent
A2-004 was reviewed by 001. Level: Excellent Excellent Competent Competent
A2-004 was reviewed by 002. Level: Excellent Excellent Competent Competent
A2-004 was reviewed by 003. Level: Competent Competent Competent Competent
A2-003 was reviewed by 001. Level: Excellent Excellent Competent Competent
A2-003 was reviewed by 002. Level: Excellent Excellent Competent Competent
A2-003 was reviewed by 004. Level: Competent Competent Competent Competent
A2-002 was reviewed by 001. Level: Excellent Excellent Competent Competent
A2-002 was reviewed by 003. Level: Excellent Excellent Competent Competent
A2-002 was reviewed by 004. Level: Competent Competent Competent Competent
A2-001 was reviewed by 002. Level: Excellent Excellent Competent Competent
A2-001 was reviewed by 003. Level: Excellent Excellent Competent Competent
A2-001 was reviewed by 004. Level: Competent Competent Competent Competent
Assignment: A1, Criterion: Knowledge/Understanding, AvgScore: 2.7
Assignment: A1, Criterion: Thinking/Inquiry, AvgScore: 2.7
Assignment: A1, Criterion: Communication, AvgScore: 2.0
Assignment: A1, Criterion: UseOfVisualAids, AvgScore: 2.0
Assignment: A1, Student: 001, Score: 2.3
Assignment: A1, Student: 002, Strength: Knowledge/Understanding Thinking/Inquiry
Assignment: A1, Student: 003, Weakness: Communication UseOfVisualAids
Assignment: A2, Criterion: Knowledge/Understanding, AvgScore: 2.7
Assignment: A2, Criterion: Thinking/Inquiry, AvgScore: 2.7
Assignment: A2, Criterion: Communication, AvgScore: 2.0
Assignment: A2, Criterion: UseOfVisualAids, AvgScore: 2.0
Assignment: A2, Student: 004, Score: 2.3
Assignment: A2, Student: 002, Strength: Knowledge/Understanding Thinking/Inquiry
Assignment: A2, Student: 001, Weakness: Communication UseOfVisualAids